_articles/getting-started/how-to-change-your-master-password.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: article
+title: How to change your master password
+category: Getting Started
+featured: false
+popular: false
+tags: [password, account]
+---
+
+The master password can only be changed from the [web](https://vault.bitwarden.com) vault.
+
+1. Log in to the [web vault](https://vault.bitwarden.com)
+2. Click **Settings** on the sidebar 
+3. Click **Change Master Password** under the **Master Password** section
+4. Type in your current password, the new password you want, and then re-type to confirm
+5. Click **Change Password**
```

Comment:
Could have left the sub point if you wanted, i was just suggesting to remove the HTML from it. I'm fine either way.